### DISCUSSION TITLE
Always in state "Connecting..."

### DISCUSSION THREAD

**Discussion by rowi-net** [bisand/node-red-contrib-tibber-api#96] [Q&A]
node-red-contrib-tipper-api worked fine for a quite long time. Now the tibber feed node mostly shows "Connecting..." as it obviously tries to exchange credentials with https://api.tibber.com/v1-beta/gql. I checked my Access Token, but this is also ok. 

Is there a way to debug the connection attempts a bit further? Maybe the tibber servers are down and I try to find the solution on my own system..

Best,
Rolf

**Comment by Thommythekid69** [comment]
Hi,
same problem here. I used node-red-contrib-tibber-api 6.0.4 for a pretty long time to monitor my tibber consumption and production and it worked until 3.9.24 well. From then I observed that the node in node red is stuck in state "connecting". I have doublechecked the API token and home ID. Everything is working from https://developer.tibber.com/explorer but not from the node red node. 

kind regards
Thommythekid

**Reply by rowi-net** [reply]
In the end I fully removed the plugin, restarted the machine and re-installed the plugin again. Then it worked again.

**Reply by Thommythekid69** [reply]
Great ! This also worked for me !!!   Thank you so much
